htdocs/compta/bank/bankentries_list.php

```diff
@@ -1462,12 +1457,12 @@
 		// Action column
 		if (getDolGlobalString('MAIN_CHECKBOX_LEFT_COLUMN')) {
 			print '<td>';
-			if ($massactionbutton || $massaction) {   // If we are in select mode (massactionbutton defined) or if we have already selected and sent an action ($massaction) defined
-				$selected = 0;
-				if (in_array($obj->rowid, $arrayofselected)) {
-					$selected = 1;
+			if (!$objp->conciliated && $action == 'reconcile') {
+				if ($objp->num_releve) {
```

Comment:
We can remove this 3 lines (it was used before to ad a space between the checkbox and the number. They are on 2 different columns now.